pkg/operator/ceph/config/datapath_test.go

```diff
@@ -26,70 +26,70 @@ func TestNewStatefulDaemonDataPathMap(t *testing.T) {
 	// mon
 	d := NewStatefulDaemonDataPathMap("/var/lib/rook", "/mon-a/data", MonType, "a", "rook-ceph")
 	assert.Equal(t, &DataPathMap{
-		HostDataDir:       "/var/lib/rook/mon-a/data",
-		ContainerDataDir:  "/var/lib/ceph/mon/ceph-a",
-		ContainerLogDir:   "/var/log/ceph",
-		HostLogDir:        "/var/lib/rook/rook-ceph/log",
-		HostCrashDir:      "/var/lib/rook/rook-ceph/crash",
-		ContainerCrashDir: "/var/lib/ceph/crash",
+		HostDataDir:      "/var/lib/rook/mon-a/data",
+		ContainerDataDir: "/var/lib/ceph/mon/ceph-a",
+		// ContainerLogDir:   "/var/log/ceph",
+		HostLogDir:   "/var/lib/rook/rook-ceph/log",
```

Comment:
This would be `HostLogAndCrashDir` I think, but it seems like you just haven't gotten to this code yet.